Mathlib/LinearAlgebra/SymmetricAlgebra/MvPolynomial.lean

```diff
@@ -0,0 +1,80 @@
+/-
+Copyright (c) 2025 Raphael Douglas Giles. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Raphael Douglas Giles, Zhixuan Dai, Zhenyan Fu, Yiming Fu, Jingting Wang
+-/
+import Mathlib.LinearAlgebra.SymmetricAlgebra.Basic
+import Mathlib.LinearAlgebra.Dimension.Basic
+import Mathlib.RingTheory.MvPolynomial
+
+/-!
+# Main Definitions
+
+1. `SymmetricAlgebra.equivMvPolynomial` give an algebra isomorphism between symmetric algebra over a
+  free module and multivariate polynomial over the basis, which is analogous to
+  `TensorAlgebra.equivFreeAlgebra`.
```

Comment:
I think this would be more useful in the `/--` docstring below, rather than here.